### DISCUSSION TITLE
Authelia release

### DISCUSSION THREAD

**Discussion by james-d-elliott** [authelia/authelia#6696] [General]
What information is available about the upcoming release?

**Comment by glime-ai** [comment]
The upcoming Authelia version 4.38 is highlighted as a significant release with several new features and improvements to existing ones. This version is described as one of the biggest releases to date for Authelia, adding several major roadmap items. Despite the anticipation for new features, the development team has taken a longer time than usual for this release, aiming to ensure a high-quality user experience by adding automated tests for new features in both the backend and frontend. This approach includes automated browser-based testing in Chromium.

For this larger release, the team is considering asking users to help with some experimentation, especially those who are comfortable backing up their database. This suggests a proactive approach to quality assurance and user involvement in the pre-release phase.

An important note from the pre-release notes is that some changes in this release will deprecate older configurations. While these changes are intended to be backwards compatible, users are advised to make necessary adjustments to their configurations to access new features or enhancements that may not be available or possible without these adjustments. Guides and an FAQ for making these adjustments are planned to be published on the blog.

The pre-release notes also mention the availability of pre-production builds of 4.38.0, specifically mentioning a beta3 version that includes most, if not all, final features. Feedback is being sought through GitHub Discussions, indicating an open and community-involved development process.

Lastly, the release notes include links to various integration and configuration guides, suggesting that the release will encompass a wide range of features and improvements across different aspects of Authelia, including Kubernetes integration, proxy integration, LDAP configuration, and session management among others.

---

Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

This comment was generated by [Glime](https://glimelab.ai/).

**Reply by Jason1roy** [reply]
Thank you for sharing the pre-release notes for Authelia version 4.38! It's exciting to hear about the significant improvements and new features coming with this release. I appreciate the development team's dedication to ensuring a high-quality user experience by investing time in automated testing, especially with browser-based testing in Chromium.
